package.json

```diff
@@ -32,56 +32,55 @@
     "start": "node scripts/build.js --bundle --serve",
     "build": "node scripts/build.js --bundle --types --copydir \"docs/dist\"",
-    "prepublishOnly": "npm run build && npm run test",
+    "release": "npm run spellcheck && npm run prettier:check && npm run ts-check && npm run build && npm run test",
+    "prepublishOnly": "npm run release",
     "prettier": "prettier --write --loglevel warn .",
-    "create": "plop --plopfile scripts/plop/plopfile.cjs",
+    "prettier:check": "prettier --check --loglevel warn .",
+    "ts-check": "tsc --noEmit",
+    "create": "plop --plopfile scripts/plop/plopfile.js",
     "test": "web-test-runner \"src/**/*.test.ts\" --node-resolve --playwright --browsers chromium firefox webkit",
     "test:watch": "web-test-runner \"src/**/*.test.ts\" --node-resolve --playwright --browsers chromium firefox webkit --watch",
-    "spellcheck": "cspell \"**/*.{js,ts,jsx,tsx,json,html,xml,css,scss,sass,md}\""
+    "spellcheck": "cspell \"**/*.{js,ts,jsx,tsx,json,html,xml,css,scss,sass,md}\" --no-progress",
+    "list-outdated-dependencies": "npm-check-updates --format repo --peer",
+    "update-dependencies": "npm-check-updates --peer -u && npm install && npm run prettier && npm run release"
   },
   "dependencies": {
     "@lit-labs/react": "^1.0.1",
-    "@popperjs/core": "^2.7.0",
+    "@popperjs/core": "^2.11.2",
     "@shoelace-style/animations": "^1.1.0",
     "@shoelace-style/localize": "^2.1.3",
     "color": "4.1",
     "lit": "^2.0.2",
     "qr-creator": "^1.0.0"
   },
   "devDependencies": {
-    "@custom-elements-manifest/analyzer": "^0.4.12",
-    "@open-wc/testing": "^2.5.33",
-    "@types/color": "^3.0.1",
-    "@types/react": "^17.0.33",
-    "@web/dev-server-esbuild": "^0.2.12",
-    "@web/test-runner": "^0.13.5",
+    "@custom-elements-manifest/analyzer": "^0.5.7",
+    "@open-wc/testing": "^3.0.3",
+    "@types/color": "^3.0.2",
+    "@types/react": "^17.0.38",
+    "@web/dev-server-esbuild": "^0.2.16",
+    "@web/test-runner": "^0.13.23",
     "@web/test-runner-playwright": "^0.8.8",
-    "bluebird": "^3.7.2",
     "bootstrap-icons": "^1.7.2",
-    "browser-sync": "^2.26.14",
-    "chalk": "^4.1.0",
+    "browser-sync": "^2.27.7",
+    "chalk": "^5.0.0",
     "command-line-args": "^5.2.0",
-    "comment-parser": "^1.1.5",
-    "concurrently": "^5.3.0",
+    "comment-parser": "^1.3.0",
     "cspell": "^5.14.0",
     "del": "^6.0.0",
     "download": "^8.0.0",
-    "esbuild": "^0.12.4",
+    "esbuild": "^0.14.10",
     "front-matter": "^4.0.2",
-    "get-port": "^5.1.1",
-    "globby": "^11.0.4",
-    "husky": "^4.3.8",
-    "lint-staged": "^12.1.4",
+    "get-port": "^6.0.0",
+    "globby": "^12.0.2",
+    "husky": "^7.0.4",
+    "lint-staged": "^12.1.5",
     "lunr": "^2.3.9",
-    "mkdirp": "^0.5.5",
-    "plop": "^2.7.4",
+    "npm-check-updates": "^12.1.0",
+    "pascal-case": "^3.1.2",
+    "plop": "^3.0.5",
     "prettier": "^2.5.1",
     "react": "^17.0.2",
-    "recursive-copy": "^2.0.11",
```

Comment:
This is still being used in `scripts/build.js` and `scripts/make-icons.js`.

Reply:
That's very weird. When I look at this commit in isolation it shows that I'm just updating these packages and not removing them. Not sure what happened (maybe in a rebase something went wrong).